### ISSUE TITLE
Royalty Dependency

### ISSUE THREAD

**Issue by vegapnk** [vegapnk/RJW-Genes#89]
There have been some angry people on discord, saying that due to some changes it requires Royalty now. 
Let's find out why / and what is happening. 

Likely there is just a small thing that slipped through. 
v2.1.0-beta-1, but I think it's also in v.2.0

Example Issue 1:
```
Failed to find any textures at Things/Building/Ruins/AncientMechGestator while constructing Multi(initPath=Things/Building/Ruins/AncientMechGestator, color=RGBA(1.000, 1.000, 1.000, 1.000), colorTwo=RGBA(1.000, 1.000, 1.000, 1.000))
UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Log.Error_Patch1 (string)
Verse.Graphic_Multi:Init (Verse.GraphicRequest)
Verse.GraphicDatabase:GetInner<Verse.Graphic_Multi> (Verse.GraphicRequest)
Verse.GraphicDatabase:Get (Verse.GraphicRequest)
Verse.GraphicDatabase:Get (System.Type,string,UnityEngine.Shader,UnityEngine.Vector2,UnityEngine.Color,UnityEngine.Color,Verse.GraphicData,System.Collections.Generic.List`1<Verse.ShaderParameter>,string)
Verse.GraphicData:Init ()
Verse.GraphicData:get_Graphic ()
Verse.ThingDef:<PostLoad>b__379_0 ()
Verse.LongEventHandler:ExecuteToExecuteWhenFinished ()
Verse.LongEventHandler:UpdateCurrentAsynchronousEvent ()
Verse.LongEventHandler:LongEventsUpdate (bool&)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Root.Update_Patch1 (Verse.Root)
Verse.Root_Entry:Update ()
```

Example Issue 2:
```
Could not load UnityEngine.Texture2D at Things/Building/Ruins/AncientExostrider/AncientExostriderCannon in any active mod or in base resources.
UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Log.Error_Patch1 (string)
Verse.ContentFinder`1<UnityEngine.Texture2D>:Get (string,bool)
Verse.Graphic_Single:Init (Verse.GraphicRequest)
Verse.GraphicDatabase:GetInner<Verse.Graphic_Single> (Verse.GraphicRequest)
Verse.GraphicDatabase:Get (Verse.GraphicRequest)
Verse.GraphicDatabase:Get (System.Type,string,UnityEngine.Shader,UnityEngine.Vector2,UnityEngine.Color,UnityEngine.Color,Verse.GraphicData,System.Collections.Generic.List`1<Verse.ShaderParameter>,string)
Verse.GraphicData:Init ()
Verse.GraphicData:get_Graphic ()
Verse.ThingDef:<PostLoad>b__379_0 ()
Verse.LongEventHandler:ExecuteToExecuteWhenFinished ()
Verse.LongEventHandler:UpdateCurrentAsynchronousEvent ()
Verse.LongEventHandler:LongEventsUpdate (bool&)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Root.Update_Patch1 (Verse.Root)
Verse.Root_Entry:Update ()
```

Example Issue 3:
```
Could not load UnityEngine.Texture2D at HLInfoIcon in any active mod or in base resources.
UnityEngine.StackTraceUtility:ExtractStackTrace ()
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.Log.Error_Patch1 (string)
Verse.ContentFinder`1<UnityEngine.Texture2D>:Get (string,bool)
HugsLib.Core.HugsLibTextures:.cctor ()
HugsLib.Quickstart.QuickstartController:DrawDebugToolbarButton (Verse.WidgetRow)
(wrapper dynamic-method) MonoMod.Utils.DynamicMethodDefinition:Verse.DebugWindowsOpener.DrawButtons_Patch0 (Verse.DebugWindowsOpener)
Verse.ImmediateWindow:DoWindowContents (UnityEngine.Rect)
Verse.Window:InnerWindowOnGUI (int)
UnityEngine.GUI:CallWindowDelegate (UnityEngine.GUI/WindowFunction,int,int,UnityEngine.GUISkin,int,single,single,UnityEngine.GUIStyle)
```

There are atleast 100 of these.

**Comment by vegapnk** [comment]
Update: This happens also with the `main` at 2.0.0 and is not a beta issue.

**Comment by vegapnk** [comment]
Jaaldabaoth says it comes from the Implants: 
```
<recipeMaker>
      <researchPrerequisite>BrainWiring</researchPrerequisite>
    </recipeMaker>
    ```

**Comment by vegapnk** [comment]
Was fixed with #91
